socs/agents/suprsync/agent.py

```diff
@@ -268,9 +268,10 @@ def make_parser(parser=None):
         '--db-pool-max-overflow', type=int, default=10,
         help="Number of connections to allow in the overflow pool."
     )
-    pgroup.add_argument('--chmod', type=str, default="g+rwX",
+    pgroup.add_argument('--chmod', type=str, default="g+rwX,o+rX",
                         help="Comma-separated chmod strings to apply to file permissions "
-                             "on transfer. Defauls to making sure files are group-writeable.")
+                             "on transfer. Defauls to making sure files are group-writeable "
```

Comment:
*Defaults

Yup, just doing my part to keep the code up to the highest standards possible.

Reply:
Heh, thanks. Resolved in https://github.com/simonsobs/socs/pull/873/commits/750f1e1b103a36ad3b34a719f0f2acf0a705eed7.